fem/gslib.hpp

```diff
@@ -231,6 +228,8 @@ class FindPointsGSLIB
        #gsl_mfem_elem   Element ids corresponding to MFEM-mesh where the points
                         were found. #gsl_mfem_elem != #gsl_elem for simplices
                         Defaults to 0 for points that were not found.
+       #gsl_steps       Optional number of steps for the local solve to find a
```

Comment:
I think this is good for debugging etc but not sure if we need to actually add this to the interface.